### ISSUE TITLE
Support scheduled tasks

### ISSUE THREAD

**Issue by chdsbd** [denoland/deploy_feedback#122]
It would be great if I could schedule a project to run on a schedule.

**Comment by ry** [comment]
Yes, we want to do this. We are researching support for cron jobs and/or background workers. But no estimate yet of when this will be supported.

**Comment by ericlery** [comment]
Any update on this? Like the closed #224 mentioned, CloudFlare Workers are a good way to implement this, I would love a native Deno Deploy integration.

**Comment by isaced** [comment]
Yes, it would be great if Deno Deploy could support cron jobs, i am looking forward to it. 🫶

**Comment by shevernitskiy** [comment]
Still waiting, still hope:)

**Comment by jonathanstanley** [comment]
related: https://github.com/denoland/deploy_feedback/issues/143#issuecomment-997841243

**Comment by lloydjatkinson** [comment]
Has this been considered any further?

**Comment by Smidgens** [comment]
I would love this feature. It'd save me all the tedium of having to manage yet another cloud dashboard just to configure a worker that pings my Deno app periodically.

**Comment by Jolg42** [comment]
Crons are now in Beta! I tried them and it works. With Crons, [KV](https://deno.com/kv), and [Queues](https://deno.com/blog/queues) it's getting really exciting 🎊 

Here is the blog post I read where they announced it:
https://deno.com/blog/cron
https://news.ycombinator.com/item?id=38461121

Docs: https://docs.deno.com/kv/manual/cron

Note that they are actively seeking feedback for Deno Cron:
>  If you’re using it (or plan to use it) in a commercial capacity and want to connect directly to one of our engineers for technical support, please let us know [via email](mailto:deploy@deno.com) or [Discord](https://discord.gg/deno).
